### PR TITLE
change branch-alias version from 2.7 to 2.8

### PR DESCRIPTION
**Pull request by Gemorroj**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "bin": ["bin/doctrine"],
     "extra": {
         "branch-alias": {
-            "dev-master": "2.7.x-dev"
+            "dev-master": "2.8.x-dev"
```
Comment (greg0ire) [inline composer.json:56]: The alias of the master branch is not 2.8 but 3.0.

Comment (Gemorroj) [reply composer.json:56]: this pr for 2.8.x branch. master have correct alias.

Comment (greg0ire) [reply composer.json:56]: What's the point of changing the alias of the `master` branch in a PR to another branch?

https://getcomposer.org/doc/articles/aliases.md#branch-alias

Comment (Gemorroj) [reply composer.json:56]: dont understan you.
We have https://github.com/doctrine/orm/tree/2.7.x branch with alias `2.7.x-dev` (https://github.com/doctrine/orm/blob/2.7.x/composer.json#L55)
For https://github.com/doctrine/orm/tree/2.8.x branch i guess alias should be `2.8.x-dev`
For https://github.com/doctrine/orm/tree/master `3.0.x-dev` now

Comment (greg0ire) [reply composer.json:56]: > For https://github.com/doctrine/orm/tree/master 3.0.x-dev now

I think that's the only one that matters, as suggested in the docs I linked above.

> The alias must be a comparable dev version, and the branch-alias must be present on the branch that it references. For dev-master, you need to commit it on the master branch.

The alias could be removed from other branches for all I care.

**Comment by greg0ire** [comment]
We don't use branch aliases anymore.

**Pull request closed without merging**